middleware/opencensus_tracing.go

```diff
@@ -2,68 +2,90 @@ package middleware
 
 import (
 	"context"
+	"crypto/rand"
 	"encoding/base64"
 	"fmt"
+	"math/big"
 	"net/http"
+	"strconv"
 
+	"github.com/go-chi/chi/v5"
 	"go.opencensus.io/trace"
 	"go.opencensus.io/trace/propagation"
 )
 
-const HeaderNameOpencensusSpan = "X-Opencensus-Span"
+const (
+	headerNameOpencensusSpan           = "X-Opencensus-Span"
+	headerNameOpencensusSpanEventIDKey = "X-Opencensus-Event-ID"
+	spanRequestPayloadAttributeKey     = "request_payload"
+	spanResponsePayloadAttributeKey    = "response_payload"
+	payloadSizeLimit                   = 1024
```

Comment:
I guess that we can align that value with quotas for stackdriver: https://cloud.google.com/trace/docs/quotas

Reply:
https://github.com/krzysztofreczek/chi-opencensus-tracing/pull/1/commits/944c4fb23efc7f42ac1923b6ded1364bab34e21a